### DISCUSSION TITLE
Referendum Application and Protest in Metaverse

### DISCUSSION THREAD

**Discussion by adamhara** [WeAreMahsaAmini/Core#110] [Ideas]
I think one of the problems is that everyone asks how many people we have. It would be very helpful if we could get a statistic that shows how many people oppose this system. In the future, we may even be able to create a country in Metaverse. Next, let's put the model into practice in a real-world environment, like a development or production environment

**Comment by AnonymousMaximus2** [comment]
The idea is great but

Creating a Metaverse nation needs much money, 

to gather the money, 

you need infrastructure for this idea,

and I explained this infrastructure here:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Reply by adamhara** [reply]
That's great, we can take small steps to get started

**Comment by ghost** [comment]
But if you protest in the metaverse, only the people in the metaverse are going to see it. The whole point of protesting is for people not involved in the protest to hear your voice.
